### DISCUSSION TITLE
Drilldown

### DISCUSSION THREAD

**Discussion by SakshiTharkude** [apache/superset#20188] [Ideas]
This feature drilldown would make superset more flexible like power bi and tableau.
Dashboards would appear to look more interactive.

**Comment by rusackas** [comment]
This is definitely a feature that multiple orgs have been discussing, and some loose plans are starting to come together into more solid ones. Anyone interested is welcome to join the #drill-actions channel on the Superset Slack workspace.

**Reply by SakshiTharkude** [reply]
Could you please tell me in detail 
In which visualization type where can i find this option of emit filter.
And is it similar like drilldown ?
